src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -709,15 +709,14 @@ private function getOrderPrices(Order $order): OrderPricesForViewing
             $wrappingPrice = (float) $order->total_wrapping_tax_excl;
             $shippingPrice = (float) $order->total_shipping_tax_excl;
             $shippingRefundable = max(0, $shipping_refundable_tax_excl);
-            $totalAmount = (float) $order->total_paid_tax_excl;
         } else {
             $productsPrice = (float) $order->total_products_wt;
             $discountsAmount = (float) $order->total_discounts_tax_incl;
             $wrappingPrice = (float) $order->total_wrapping_tax_incl;
             $shippingPrice = (float) $order->total_shipping_tax_incl;
             $shippingRefundable = max(0, $shipping_refundable_tax_incl);
-            $totalAmount = (float) $order->total_paid_tax_incl;
         }
+        $totalAmount = (float) $order->total_paid_tax_incl;
```

Comment:
Hi @Progi1984 I suppose the PR is in WIP but this seems to be something that needs to be explained 😅  why always return total_paid_tax_incl ?

Reply:
I will link tomorrow if @Mathshir has done improvements on specs

Reply:
I also don't understand this, we're supposed to switch the display with/without taxes but you always use the amount with tax 🤔